uint8 idle = 0
uint8 configuring = 1
uint8 speaking = 2
uint8 listening = 3
uint8 waiting = 4
uint8 dialogue_state